kuma/static/styles/components/structure/document-head.scss

```diff
@@ -6,6 +6,7 @@
     padding: $first-content-top-padding 0 $last-content-bottom-padding 0;
 }
 
+div[class*='titlebar'],
```

Comment:
Looks like we have classes with names ending in `titlebar` and `titlebarContainer`, and this selects both of them.

1. Is that the expected targeting?
2. Is this change necessary for this issue? I'm not sure understand its the relevance to screen readers.

Reply:
This is one of the big reasons I am also working on moving most of the styles away from emotion and back into Sass. Those generated class names that start with `css-randomness-class-name` is forcing one to use really expensive CSS selectors.

Also in general, styles should only be tightly coupled with a component if they are truly unique to the component. For right now, this thankfully does not have unintended consequences(that I noticed).